CMakeLists.txt

```diff
@@ -451,7 +451,7 @@ IF (WIN32)
     SET (DEFAULT_BIN_SUBDIR bin)
     SET (DEFAULT_CGIBIN_SUBDIR bin)
     # put all the build products into a single directory
-    # under build (doesnt affect install target) to make for
+    # under build (doesn't affect install target) to make for
```

Comment:
> than something which doesn't exist.
Effectively yes, for the rest I don't know how it's accepted or not then don't worry...